scalableminds/webknossos
# Enable wrap-in-group option when importing single NML

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Drag single NML into tracing view, you should still have the option to wrap its trees in a group

### Issues:
- fixes #4486 

------
